docs/en/index.rst

```diff
@@ -846,11 +846,13 @@ beginning::
     // in tests/bootstrap.php
     use Migrations\TestSuite\Migrator;
 
+    $migrator = new Migrator();
+
     // Simple setup for with no plugins
-    Migrator::migrate();
+    $migrator->run();
 
     // Run migrations for multiple plugins
-    Migrator::migrate([
+    $migrator->run([
```

Comment:
We'll need to update the app skeleton, and testing docs again.

Reply:
https://github.com/cakephp/docs/pull/7132

Reply:
https://github.com/cakephp/app/pull/856